java/offline-geocode/src/main/java/com/esri/arcgisruntime/sample/offlinegeocode/MainActivity.java

```diff
@@ -363,241 +325,90 @@ protected void onResume() {
 
   @Override
   protected void onDestroy() {
-    super.onDestroy();
     mMapView.dispose();
+    super.onDestroy();
   }
 
+  /**
+   * Define a listener to handle drag events.
+   */
   private class DragTouchListener extends DefaultMapViewOnTouchListener {
 
-    float dX, dY;
-
-    public DragTouchListener(Context context, MapView mapView) {
+    DragTouchListener(Context context, MapView mapView) {
       super(context, mapView);
     }
 
     @Override
     public boolean onTouch(View view, MotionEvent event) {
```

Comment:
Looks like onTouch function is not needed. It is not called from anywhere in the code.